moflask/tests/jwt_test.py

```diff
@@ -77,3 +77,13 @@ def test_anonymous_session_with_optional(protected_app):
     assert response.json["identity"] is None
     assert response.json["user_claims"]["org"] is None
     assert response.json["user_claims"]["roles"] == []
+
+
+def test_anonymous_session_with_organization(protected_app):
+    """Test that sessions for anonymous requests read the X-ISt-Org header."""
+    with protected_app.test_client() as client:
+        response = client.get("/optional", headers={"X-ISt-Org": "test-org"})
```

Comment:
Since HTTP headers are case insensitive, I guess there is no "correct" way of writing the new header, but `X-ISt-Org` looks very weird to me? The server part of this feature went for all upper case `X-IST-ORG`.

Reply:
Ok, will change it. I don’t mind. But strictly speaking I would wonder what T, R and G are standing for in this acronym? Also driving design choices by what is committed into ansible without review seems a bit weird. Originally I wanted to go for `X-Impact-Stack-Org`.

Reply:
I’ve now changed it to a header name with completely irregular casing to showcase that the case doesn’t matter.

Reply:
At least not when it comes to Impact Stack it seems :see_no_evil: